modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/checkpoint/CheckpointManager.java

```diff
@@ -136,15 +136,17 @@ public CheckpointManager(
 
         FileIOFactory ioFactory = persistenceCfg.getFileIOFactory();
 
+        CheckpointReadWriteLock lock = new CheckpointReadWriteLock(logger);
```

Comment:
Is this lock created only to pass it to method?

Reply:
No it's a checkpoint lock, it's also used by checkpointer